Update/yandere custom armour

OverYandere opened the pull request:
Update to both my own armour from the demo versions, and to how specialty custom helmets are handled.

Lupus590 reviewed (commented):
Your ODST helmets have changed?
Also, I want to point out, the different camos nolonger match in customisation for your ODST helmets. I'm not sure if that is intended or not so just highlighting it.

OverYandere commented:
> Your ODST helmets have changed? Also, I want to point out, the different camos nolonger match in customisation for your ODST helmets. I'm not sure if that is intended or not so just highlighting it.

I added a flag and changed around some text. But the colors shouldn't be noticeably different. I will check on my end if the odst helmets are not matching the armor.
